features/networking/multus.feature

```diff
@@ -1244,3 +1244,33 @@ Feature: Multus-CNI related scenarios
       | /usr/sbin/ip | a  |
     Then the output should contain:
       | 192.168.22.101    |
+
+  # @author weliang@redhat.com
+  # @case_id OCP-28518
+  @admin
+  Scenario: Multus custom route change with route override	
+    # Make sure that the multus is enabled
+    Given the multus is enabled on the cluster
+    Given the default interface on nodes is stored in the :default_interface clipboard
+
+    # Create the net-attach-def via cluster admin
+    Given I have a project
+    When I run oc create as admin over "<%= BushSlicer::HOME %>/testdata/networking/multus-cni/NetworkAttachmentDefinitions/route-override.yaml" replacing paths:
+      | ["metadata"]["namespace"]  | <%= project.name %> | 
+      | ["metadata"]["name"]       | route-override      |
+      | ["spec"]["config"] | '{ "cniVersion": "0.3.0", "name" : "mymacvlan", "plugins": [ { "type": "macvlan", "master": "ens3", "mode": "bridge", "ipam": { "type": "static", "addresses": [{"address": "192.168.20.2/24"}] } }, { "type" : "route-override", "addroutes": [ { "dst": "192.168.10.0/24" }] } ] }' |
```

Comment:
table alignment issue...minor

Reply:
`"master": "ens3"` here `ens3` should be replace with `default_interface`
does `"master" ` is must parameter? if not. I suggest delete this since we cannot sure the node pod scheduled has the `default_interface`. The step line `1254` get the interface in one of worker.  but primary interface between worker maybe different( rhcos worker and rhel worker).  eg. line `1254` got the interface is `ens3`.   however the pod is scheduled to rhel worker which primary interface is `eth0`. then the pod should be failed to created.
the `master` parameter should be a optional parameter from 4.2 I remembered.

Reply:
@pruan-rht Fixed table alignment issue, thanks!
@zhaozhanqi Remove "master" from this test case, thanks!